test/MSVC/MSVC_USE_SCRIPT.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
Can you update the header copyright blurb with the same content from templates/Tests.py

That has the updated copyright blurb.

Reply:
I'll fix and push.